main_supcon.py

```diff
@@ -422,18 +484,28 @@ def main():
             loss = train_category(train_loader, model, criterion, optimizer, epoch, opt)
         else:
             loss = train(train_loader, model, criterion, optimizer, epoch, opt)
+        if opt.wandb:wandb.log({'loss':loss}, step=epoch)
```

Comment:
python style guides says do not put if statements on one line. Change this everywhere https://peps.python.org/pep-0008/